sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusConnection.cs

```diff
@@ -69,18 +71,19 @@ internal class ServiceBusConnection : IAsyncDisposable
         /// </summary>
         ///
         /// <param name="connectionString">The connection string to use for connecting to the Service Bus namespace.</param>
-        /// <param name="options">A set of options to apply when configuring the connection.</param>
-        ///
+        /// <param name="client"></param>
         /// <remarks>
         ///   If the connection string is copied from the Service Bus entity itself, it will contain the name of the desired Service Bus entity,
         ///   and can be used directly without passing the  name="entityName" />.  The name of the Service Bus entity should be
         ///   passed only once, either as part of the connection string or separately.
         /// </remarks>
         internal ServiceBusConnection(
             string connectionString,
-            ServiceBusClientOptions options)
+            ServiceBusClient client)
```

Comment:
I have the same hesitation here that I mentioned in the transport client; this changes the flow of dependencies from a single direction into circular.   Have we considered surfacing the connected/disconnected events from the transport client through the connection?